### PR TITLE
Upgrade go version

### PR DESCRIPTION
**Pull request by psav**
Upgrade go version

### CODE REVIEW

File: .tekton/clowder-pull-request.yaml [hunk 1]
```diff
@@ -244,6 +244,97 @@ spec:
...
+        - name: ociStorage
+          value: $(params.output-image).make-template-kustomize
+        - name: IMAGE_EXPIRES_AFTER
+          value: $(params.image-expires-after)
+      runAfter:
+        - prefetch-dependencies
+      taskSpec:
+        results:
+        - name: TEMPLATE_ARTIFACT
+          type: array
```
Comment (ernesgonzalez33) [inline .tekton/clowder-pull-request.yaml:261]: Why is this an array?

File: .tekton/clowder-pull-request.yaml [hunk 2]
```diff
@@ -244,6 +244,97 @@ spec:
...
+      runAfter:
+        - make-template
+      taskSpec:
+        params:
+          - description: The Trusted Artifact URI pointing to the artifact with the application source code.
+            name: SOURCE_ARTIFACT
+            type: string
+          - description: The Trusted Artifact URI pointing to the artifact with the template.
+            name: TEMPLATE_ARTIFACT
+            type: string
```
Comment (ernesgonzalez33) [inline .tekton/clowder-pull-request.yaml:311]: And here a string?

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/setup-go@v4
         name: Set up Go 1.x
         with:
-          go-version: "1.21.9"
+          go-version: "1.22.7"
```
Comment (maskarb) [inline .github/workflows/lint.yml:25]: You could use `go-version-file: 'path/to/go.mod'` instead. It's one less thing to remember to update in a future go version update.

We use it [here](https://github.com/project-koku/koku-metrics-operator/blob/main/.github/workflows/ci.yaml#L20-L23).

Comment (psav) [reply .github/workflows/lint.yml:25]: Thanks for that

**Review by gburges — APPROVED** [review]

**Pull request merged**